tests/basic/storage_pool_test.go

```diff
@@ -9597,16 +9601,18 @@ func scheduleApps() []*scheduler.Context {
 	contexts := make([]*scheduler.Context, 0)
 	for i := 0; i < Inst().GlobalScaleFactor; i++ {
 		log.Infof("Deploy app %v", i)
-		contexts = append(contexts, ScheduleApplications(fmt.Sprintf("pooltest-%d", i))...)
+		contexts = append(contexts, ScheduleApplicationsOnNamespace(
+			testingAppNamespace, fmt.Sprintf("pooltest-%d", i))...)
```

Comment:
A static namespace for testing app(s) allows a faster debugging experience when the need: instead of listing all namespaces and copy the one with randomness in the naming, we can directly pass `-n testing-apps` to collect information. The same approach is used for torpedo and porx deployments. 

What's the use case for introducing randomness in the namespace to deploy a testing app?

Reply:
for the system test we deploy multiple apps, deploying all the apps in the namespace will make it difficult to triage in case of errors of particular app. 
Also segregating based on namespace will help in few scenarios where we want only particular apps to be destroyed or scaled.